plugins/UM3NetworkPrinting/src/DiscoverUM3Action.py

```diff
@@ -116,22 +117,28 @@ def setGroupName(self, group_name: str) -> None:
             # Ensure that the connection states are refreshed.
             self._network_plugin.reCheckConnections()
 
-    @pyqtSlot(str)
-    def setKey(self, key: str) -> None:
-        Logger.log("d", "Attempting to set the network key of the active machine to %s", key)
+    @pyqtSlot(QObject)
+    def setKey(self, printer_device: Optional[PrinterOutputDevice]) -> None:
```

Comment:
This shouldn't be called setKey if it doesn't accept a key...

Reply:
Renamed it to `associateActiveMachineWithPrinterDevice()` in aba3b6dd8